README.md

```diff
@@ -136,6 +136,22 @@ Please log the time taken to perform the operations described, and within reason
    * temporal accuracy to within 6 hours
 Please also remove any columns which you believe could lead to the resident who made the request being identified. We expect in the accompanying report that you will justify as to why this data is now anonymised. Please limit this commentary to less than 500 words. If your code is written in a code notebook such as Jupyter notebook or Rmarkdown, you can include this commentary in your notebook.
 
+### 6. Data Visualization Task (if applying for a Data Visualization Engineering Position)
+
+Using the `sr_hex_truncated.csv` dataset and open source web development technologies, develop a data visualization / dashboard that help to answer the question:
```

Comment:
open source web development technologies -> open source web technologies (this is a bit of a nitpick)

viz -> vis

Reply:
agree

Reply:
should we ask for an interactive visulaisation/dashboard?

Reply:
Point 3 ask: *Make (1) and (2) interactive to allow users to explore the data and uncover insights*